packages/experiment-tag/src/subscriptions.ts

```diff
@@ -373,6 +378,62 @@ export class SubscriptionManager {
     });
   };
 
+  private setupExitIntentPublisher = () => {
+    // Check if any page objects use exit_intent trigger
+    const hasExitIntentTriggers = Object.values(this.pageObjects).some(
+      (pages) =>
+        Object.values(pages).some(
+          (page) => page.trigger_type === 'exit_intent',
+        ),
+    );
+
+    if (!hasExitIntentTriggers) {
+      return;
+    }
+
+    // Get minimum time requirement (use lowest value so listener activates earliest)
+    let minTimeOnPageMs = 0;
+    for (const pages of Object.values(this.pageObjects)) {
+      for (const page of Object.values(pages)) {
+        if (page.trigger_type === 'exit_intent') {
+          const triggerValue = page.trigger_value as ExitIntentTriggerValue;
+          if (triggerValue.minTimeOnPageMs !== undefined) {
+            minTimeOnPageMs =
+              minTimeOnPageMs === 0
+                ? triggerValue.minTimeOnPageMs
+                : Math.min(minTimeOnPageMs, triggerValue.minTimeOnPageMs);
```

Comment:
it looks like you can shorten this to `minTimeOnPageMs = Math.min(minTimeOnPageMs, triggerValue.minTimeOnPageMs)`